portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,11 +53,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = getParameter(request, "text-input", "");
```

Comment:
same, except not sure what the last parameter is for here? Is that the default parameter? If so, create a default parameter global field